test/download-asset-test.js

```diff
@@ -0,0 +1,21 @@
+import test from 'tape'
+import downloadAsset from '../lib/download-asset'
+import fs from 'fs-extra'
+import path from 'path'
+var tmpDirectory = path.join(__dirname, 'tmp')
+
```

Comment:
It is great that you have test, could you add test case for failing the download